src/prefabs/dataContainer.js

```diff
@@ -25,6 +25,8 @@
       model: null,
     });
 
+    const [fetchModelId, setFetchModelId] = React.useState(null);
```

Comment:
I think we should rename it to `fetchedModelId`. The name `fetchModelId` give me the indication that it still needs to be fetched, which is not true.

Reply:
At the moment of setting the `fetchModelId` it does still need to be fetched right? I am using this state variable so we can always use this in the ModelQuery

Reply:
`fetchModelData` is maby a better name? Because we are using the UUID in this state to fetch the model's data. Agreed?

Reply:
I've we still have to fetch it, I think `fetchModelId` will be fine!